### PR TITLE
Fix services

### PR DESCRIPTION
**Pull request by jaynaxissa**
## Refactoring the services component

https://ywgc.atlassian.net/browse/YWGC-38?atlOrigin=eyJpIjoiMGYwZGU3ZjRhOWFjNGY3NDk2ZmQzNDY2NzQ5ZGJjODAiLCJwIjoiaiJ9

## Work done
- Made all the changes based on the new design on figma (add horizontal line, make sure the font size are switching to the right font size when it comes to a different screen size)

## Manual testing instructions
- Responsive to screen size.
- Make sure the padding/margin are different when it's web screen and mobile screen.
- Make sure the icons are align in the center for mobile version in columns and align in center for web screen in rows.
- All of the fonts are set up to be in a different size when it comes to a different screen size like web screen and mobile screen
- Check whether the horizontal line (underline for Services title) change its size when it's mobile. 

## Gotchas/What I learned
To be more specific in the manual testing.

### CODE REVIEW

File: src/pages/Home.jsx [hunk 1]
```diff
@@ -3,17 +3,25 @@ import BaseHeading from '../components/BaseHeading'
...
-        <BaseHeading level={1} text="Yorkshire Woods Ground Crew" />
-        <p className="mt-4 text-xl">
-          Yorkshire Woods Grounds Crew is a Detroit local lawn care and landscaping business. We specialize in lawn-mowing and clean-outs. We handle residential, business, and government properties of all sizes. We&apos;re available for one-time and ongoing engagements.
-        </p>
-        <a
-          href="tel:+123456789"
-          className="items-center w-full p-2 my-20 text-lg text-center text-white bg-green-700 rounded-lg font-body md:mt-16 hover:bg-green-600">
-          (123)-456-789
-        </a>
+    <div>
```
Comment (novellac) [inline src/pages/Home.jsx:6]: Question: Why did you add this extra `div`? It seems out of scope for the card.

File: src/components/Services.jsx [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
-        ))}
+   <div className="flex flex-col w-full md:py-24 py-6">
+     <h2 className="mx-auto text-4xl md:text-6xl text-fjalla">Services</h2>
+     <hr className='w-[180px] md:w-[720px] h-px mx-auto mt-2 mb-10 md:mb-2 bg-gray-100 border-0 rounded dark:bg-gray-700'></hr>
+    <ul className=''>
+      <li className=''>
+      {serviceData.map((data) => (
+        <ServiceItem key={data.id} data={data} />
+      ))}
+      </li>
```
Comment (novellac) [inline src/components/Services.jsx:15]: issue: This puts all the service items inside one `li`, when they should each have their own `li`. The impact of this is that screen readers and other assistive technology will assume this is a list with _one_ item, when it's really a list with more than one item.

**Review by novellac — CHANGES_REQUESTED** [review]
issue: The manual testing instructions aren't specific enough. Please let me know what I should be testing when I open the preview link. For example, should I be checking the way the section heading looks? Should I be checking the number of items? Should I be checking the placement of the icon? Etc.

**Review by alliequintano — APPROVED** [review]
Thanks for the more detailed testing instructions! Looks good to me. It might be worth getting an approval from @ajkarczewski before merging since a lot of the details for this PR have to do with the design.

**Review by ajkarczewski — APPROVED** [review]
Nice work! Looks good so far :)

**Review by novellac — APPROVED** [review]

**Pull request merged**